src/signals/incident-management/containers/IncidentDetail/components/Highlight/index.test.js

```diff
@@ -1,10 +1,50 @@
-// import React from 'react';
-// import { shallow } from 'enzyme';
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import { run } from 'test/utils';
+
+import Highlight, { HIGHLIGHT_TIMEOUT_INTERVAL } from './index';
 
-// import Highlight from 'index';
 
 describe('<Highlight />', () => {
-  it('Expect to have unit tests specified', () => {
-    expect(true).toEqual(true);
+  beforeEach(() => {
+    jest.useFakeTimers();
+  });
+
+  afterEach(() => {
+    jest.runAllTimers();
+    cleanup();
+  });
+
+  describe('rendering', () => {
+    it('should render all children that are passed in', () => {
+      const string = 'foo';
+      const { queryByTestId, queryAllByTestId } = render(
+        <Highlight subscribeTo={string}><div data-testid="highlight-child">some text</div></Highlight>
+      );
+
+      expect(queryAllByTestId('highlight-child')).toHaveLength(1);
+      expect(queryByTestId('highlight')).not.toHaveClass('highlight--active');
+    });
+  });
+
+  describe('events', () => {
+    it('should highlight the container when the value has changed and de-highlight after 3 seconds', () => {
+      const { container } = render(
+        <Highlight subscribeTo="foo"><div>some text</div></Highlight>
+      );
+
+      jest.runTimersToTime(HIGHLIGHT_TIMEOUT_INTERVAL - 1);
+      run(() => {
```

Comment:
What is the purpose of this `run` function?

Reply:
heb je de juiste branch? hij is echt weg